app/javascript/mastodon/features/compose/components/search.tsx

```diff
@@ -497,8 +497,10 @@ export const Search: React.FC<{
             <div className='search__popout__menu'>
               {recentOptions.length > 0 ? (
                 recentOptions.map(({ label, key, action, forget }, i) => (
-                  <button
+                  <div
```

Comment:
why the change in element? does `button` have additional impact on keyboard events?